src/main/java/ch/wisv/areafiftylan/utils/mail/MailServiceImpl.java

```diff
@@ -137,7 +137,7 @@ public void sendVerificationmail(User user, String url) {
                 "Please click on the following link to complete your registration: <a href=\"" + url + "\">" + url +
                         "</a><br /><br />If the link does not work, please copy the link and" +
                         " paste it into your browser.";
-        sendMail(user.getUsername(), user.getUsername(), "Confirm your registration", message);
+        sendMail(user.getUsername(), user.getProfile().getFirstName() + " " + user.getProfile().getLastName(), "Confirm your registration", message);
```

Comment:
Like Sille said: Please note that this is not possible, the first- and lastname are not set here.